### PR TITLE
Added support for new iPad Pro safe area insets

### PR DESCRIPTION
**Pull request by cesargdm**
Added support for new iPad Pro models, 11-inch and 12.9 inch (3rd generation).
Please checkup the new variables names.

See https://github.com/react-native-community/react-native-safe-area-view/issues/54

### CODE REVIEW

**Comment by brentvatne** [comment]
thanks! it's very unfortunate that we need to do this but until the layout context api lands upstream in react-native it'll work

File: index.js [hunk 1]
```diff
@@ -36,6 +40,18 @@ const isIPhoneX = (() => {
   );
 })();
 
+const isNewIPadPro = (() => {
+  if (Platform.OS === 'web') return false;
```
Comment (brentvatne) [inline index.js]: `if (Platform.OS !== 'ios')`

**Review by brentvatne — CHANGES_REQUESTED** [review]
just one small change and it's good to go

File: index.js (hunk 1 above)
Comment (cesargdm) [reply index.js]: Fixed :)

**Comment by cesargdm** [comment]
Fixed! 😊
@brentvatne

File: index.js [hunk 2]
```diff
@@ -319,7 +336,15 @@ class SafeView extends Component {
         return statusBarHeight(isLandscape);
       }
       case 'bottom': {
-        return isIPhoneX ? (isLandscape ? 24 : 34) : 0;
+        if (isIPhoneX) {
+          return isLandscape ? 24 : 34;
+        }
+
+        if (isNewIPadPro) {
+          return 20;
```
Comment (bradleyflood) [inline index.js:344]: Why `20` for iPadPro and not `34` to be the same as the iPhoneX? They both have the "Home Line" and need the same amount of clearance right? @cesargdm

Comment (bradfx) [inline index.js:344]: Why `20` for iPadPro and not `34` to be the same as the iPhoneX? They both have the "Home Line" and need the same amount of bottom clearance right? @cesargdm

Comment (bradfx) [reply index.js:344]: iPhoneX
Screen Shot 2020-04-13 at 10 01 54

iPadPro
Screen Shot 2020-04-13 at 10 02 00

**Comment by brentvatne** [comment]
@bradleyflood - i don't recommend using the old version of react-native-safe-area-view that includes this code, the latest version uses react-native-safe-area-context to correctly determine device safe areas with the platform native apis

**Pull request merged**